server/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Base image
+FROM registry.access.redhat.com/ubi8/python-36:latest
+
+USER root
```

Comment:
You probably want to check all the advice in the [guidelines](https://docs.openshift.com/container-platform/3.11/creating_images/guidelines.html) has been considered here. And maybe link to them, since a new reader may not be familiar with them.

Reply:
> If you're elevating privs to install something, please restore the excellent default of non-root user you inherited from the example image you're layering upon.
> 
> Is it `pip` here that needs root?

We have a fix for that in develop branch.